src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -14,6 +14,30 @@ const RenderMatchUp = props => {
   const [modalVisible, setModalVisible] = useState(true);
   const [teams, setTeams] = useState({});
 
+  /**
```

Comment:
Glad that you found a solution to this. Maybe there's another simpler solution that doesn't use display:none that will allow us to avoid the prop-drilling, but we can think on that more later. Good work.